test/components/CalendarMonth_spec.jsx

```diff
@@ -46,5 +48,50 @@ describe('CalendarMonth', () => {
       expect(typeof isVisible).to.equal('boolean');
       expect(wrapper.find('#month-element').exists()).to.equal(true);
     });
+
+    describeIfWindow('setMonthTitleHeight', () => {
+      const awaitChange = fn => new Promise((resolve, reject) => {
+        setTimeout(() => {
+          try {
+            fn();
+            resolve();
+          } catch (e) {
+            reject(e);
+          }
+        }, 1);
+      });
+
+      it('sets the title height after mount', () => {
+        const setMonthTitleHeightStub = sinon.stub();
+        mount(
+          <CalendarMonth
+            isVisible
+            setMonthTitleHeight={setMonthTitleHeightStub}
+          />,
+        );
+
+        return awaitChange(() => expect(setMonthTitleHeightStub.callCount).to.equal(1));
```

Comment:
```suggestion
        return awaitChange(() => expect(setMonthTitleHeightStub).to.have.property('callCount', 1));
```
this has a nicer failure message; same throughout